test/sequential/test-http-keep-alive-large-write.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const serverTimeout = common.platformTimeout(500);
+let offsetTimeout = common.platformTimeout(100);
+let socketHandle;
+let writeSize = 3000000;
+let writtenOffset = 0;
```

Comment:
If I understand this correctly this is used to check how much was written by the time we get the first `data` event. If that is correct could you rename this to something similar to `writeSizeOnFirstRead`

Reply:
It's a bit tricky to name this well. I might just add a comment. It represents the write amount of each cycle where the cycle is `write > read > timeout event`.